Author: Daniel Leidert <dleidert@debian.org>
Description: Fix linkage when using -DRDK_BUILD_THREADSAFE_SSS=ON.
 libRDBoost.so misses linkage -lboost_python if built with the above
 switch. So tests fail because of unresolved symbols in this library.
Forwarded: yes

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -157,7 +157,7 @@
 
 find_package (Threads)
 if(RDK_BUILD_THREADSAFE_SSS)
-  find_package(Boost 1.39.0 COMPONENTS thread system REQUIRED)
+  find_package(Boost 1.39.0 COMPONENTS thread system python REQUIRED)
   ADD_DEFINITIONS("-DRDK_THREADSAFE_SSS")
   set(RDKit_THREAD_LIBS ${Boost_THREAD_LIBRARY} ${Boost_SYSTEM_LIBRARY} ${CMAKE_THREAD_LIBS_INIT})
 else()
